homeassistant/components/cover/zwave.py

```diff
@@ -91,19 +91,28 @@ def current_cover_position(self):
 
     def open_cover(self, **kwargs):
         """Move the roller shutter up."""
-        self._network.manager.pressButton(self._open_id)
+        if not self._open_id:
+            self.node.set_dimmer(self.values.primary.value_id, 100)
+        else:
+            self._network.manager.pressButton(self._open_id)
 
     def close_cover(self, **kwargs):
         """Move the roller shutter down."""
-        self._network.manager.pressButton(self._close_id)
+        if not self._close_id:
+            self.node.set_dimmer(self.values.primary.value_id, 0)
+        else:
+            self._network.manager.pressButton(self._close_id)
 
     def set_cover_position(self, position, **kwargs):
         """Move the roller shutter to a specific position."""
         self.node.set_dimmer(self.values.primary.value_id, position)
 
     def stop_cover(self, **kwargs):
         """Stop the roller shutter."""
-        self._network.manager.releaseButton(self._open_id)
+        if not self._open_id:
+            self._network.manager.releaseButton(self.values.primary.value_id)
```

Comment:
Maybe any of the "Inc" "Bright" "Up" "Open" values can be used for up, so in dicovery we use any of those to give the open_id. and the same for Close. The releasebutton command is the only way to stop a level change.